### PR TITLE
Allow opting-out from announcements

### PR DESCRIPTION
**Pull request by eladb**
When we release a new version of AWS CDK, the catalog sends too many "similar" tweets, which causes Twitter to temporarily disable search against @awscdkio. In the meantime, we decided to simply ignore new @aws-cdk modules. There are sufficient people announcing them anyway. Maybe in the future we will do some aggregated tweet.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,16 +20,32 @@ The Construct Catalog will automatically discover jsii multi-language modules pu
...
 4. Publish your module to all package managers. Here are some recommended tools:
    - [jsii publish GitHub Action](https://github.com/marketplace/actions/jsii-publish) by [udondan](https://github.com/udondan)
    - [aws-delivlib](https://github.com/awslabs/aws-delivlib) used by the AWS CDK
 
-5. If your module is not picked up by @awscdkio within 10 minutes, see the troubleshooting section below.
+5. If your module is not picked up by [@awscdkio](https://twitter.com/awscdkio) within ~10 minutes, see the troubleshooting section below.
+
+`package.json` configuration:
 
+```json
```
Comment (eladb) [inline README.md]:
```suggestion
```js
```

File: packages/catalog/lib/tweeter/lambda/index.ts [hunk 2]
```diff
@@ -95,7 +97,11 @@ export async function handler(event: AWSLambda.SQSEvent, context: AWSLambda.Cont
 
     let tweetid;
 
-    if (twitter) {
+    // determine if we should announce (tweet) this module (default is true).
+    const announce = awscdkio.announce === undefined ? true : awscdkio.announce;
+    console.log(JSON.stringify({ announce }));
```
Comment (mbonig) [inline packages/catalog/lib/tweeter/lambda/index.ts]: want to keep this?

Comment (eladb) [reply packages/catalog/lib/tweeter/lambda/index.ts]: Yes, useful to add this to the log

**Comment by eladb** [comment]
> I'll take a stab at a unit test around this....

Would be awesome!

**Pull request merged**